### PR TITLE
Create style.css

### PR DESCRIPTION
**Pull request by aima98**
Adding code from To-Do-List project

### CODE REVIEW

File: modules/Structure.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/* eslint-disable class-methods-use-this */
```
Comment (tomdan-ai) [inline modules/Structure.js:1]: I believe the linters are there to guide you to JavaScript best practices, do well to enable them

**Review by tomdan-ai — COMMENTED** [review]
Hi @aima98 ,
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights
 Good commit messages ✔
 No merge conflicts ✔️
 Correct GitHub flow ✔️

### **Required Changes ♻️**
Check the comments under the review.

 Your Repo doesn't have a professional README
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: modules/Structure.js (hunk 1 above)
Comment (aima98) [reply modules/Structure.js:1]: Thank you for your feedback😄

**Review by tomdan-ai — COMMENTED** [review]
Hello @aima98  ,

Your project is complete! There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏

Congratulations! 🎉 🎉 🎉

Highlights
Professional Readme File ✔✔
Linters are setup correctly ✔✔
Clear and nice code ✔✔

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**